### PR TITLE
Adding the content on the "Intenal/devops" role and simplified scope requirements to perform API Controller operations

### PR DESCRIPTION
**Pull request by wasuradananjith**
## Purpose
Fixes: https://github.com/wso2/product-apim-tooling/issues/372

## Goals
Introduce the new role **Internal/devops** added in APIM 3.2.0 and describe the simplified scope requirements to perform API Controller operations.

## Approach
- Removed the section **Steps to Create a Custom User who can Perform API Controller Operations** n the page **API Controller/Advance Topics/Creating Custom Users to Perform API Controller Operations**.
- Modified the table under the topic ** Minimal Permissions and Scopes Required to Perform API Controller Operations** in the page **API Controller/Advance Topics/Creating Custom Users to Perform API Controller Operations**.

image

## User stories
The users can refer about the new role **Internal/devops** and the simplified scope requirements for the API Controller operations.

## Related PRs
https://github.com/wso2/carbon-apimgt/pull/8849

### CODE REVIEW

File: en/docs/learn/api-controller/advanced-topics/creating-custom-users-to-perform-api-controller-operations.md [hunk 1]
```diff
@@ -1,80 +1,17 @@
 #  Creating Custom Users to Perform API Controller Operations
 
-To perform tasks using the API Controller, a particular user must have required permissions and scopes. By default, a user with `admin` role is allowed perform all the tasks. 
+To perform tasks using the API Controller, a particular user must have required permissions and scopes. From API Manager 3.2.0 onwards, a new role named `Internal/devops` has been introduced who has the ability to perform all the API Controller related operations. You just need to create a user (Refer [Adding a new User]({{base_path}}/administer/managing-users-and-roles/managing-users/#adding-a-new-user) to learn about adding new users), assign the role `Internal/devops` and use that user to perform the API Controller operations.
```
Comment (npamudika) [inline en/docs/learn/api-controller/advanced-topics/creating-custom-users-to-perform-api-controller-operations.md:3]:
```suggestion
To perform tasks using the API Controller, a particular user must have required permissions and scopes. From API Manager 3.2.0 onwards, a new role named `Internal/devops` has been introduced who has the ability to perform all the API Controller related operations. You just need to create a new user (Refer [Adding a new User]({{base_path}}/administer/managing-users-and-roles/managing-users/#adding-a-new-user) to learn about adding new users), assign the role `Internal/devops` and use that user to perform the API Controller operations.
```

File: en/docs/learn/api-controller/advanced-topics/creating-custom-users-to-perform-api-controller-operations.md [hunk 2]
```diff
@@ -1,80 +1,17 @@
...
 
-3.  Restart the server or wait for 15 mins until the registry cache expires.
+Further, you can create your own custom user with a custom role to perform specific set of API Controller operations. Refer the below table to learn about the required permissions and scopes that are needed for each of the API Controller operation.
 
-## Minimal Permissions and Scopes Required to Perform API Controller Operations
+!!! info
+    **Steps to create a custom user with a custom role for your need** 
 
-As explained in the above section, you can create any user with a custom role to perform operations using the API Controller. But if you want that user to perform some of the API Controller operations (not all) you can refer the below table and assign the permissions/scopes accordingly for that role.
+    - As shown in [Create user roles]({{base_path}}/administer/managing-users-and-roles/managing-user-roles/#create-user-roles) section, you can create your own custom user role by assigning permissions and scopes that are required to perform a particular set of API Controller operations by reffering the table below.
```
Comment (hisanhunais) [inline en/docs/learn/api-controller/advanced-topics/creating-custom-users-to-perform-api-controller-operations.md:12]: At the end of the second line, shall we make it "referring"?

Comment (wasuradananjith) [reply en/docs/learn/api-controller/advanced-topics/creating-custom-users-to-perform-api-controller-operations.md:12]: Fixed via https://github.com/wso2/docs-apim/pull/3097/commits/74cdaa1323d12a99abd038f41a690914b8d50ab5

Comment (wasuradananjith) [reply en/docs/learn/api-controller/advanced-topics/creating-custom-users-to-perform-api-controller-operations.md:12]: Fixed via https://github.com/wso2/docs-apim/pull/3098/commits/edd8ae4c020155b3c9d8e4465a923a031e8f9472 for 3.2.0

**Pull request merged**